docs/system/k8s-metrics.md

```diff
@@ -256,6 +260,86 @@ The actual accuracy would depend on the instrumentation and operating system.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+### Metric: `k8s.node.allocatable_cpu`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.k8s.node.allocatable_cpu -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
+| -------- | --------------- | ----------- | -------------- | --------- | ------ |
+| `k8s.node.allocatable_cpu` | Gauge | `{cpu}` | Amount of cpu allocatable on the node | ![Development](https://img.shields.io/badge/-development-blue) |  |
```

Comment:
I think all of these should be `UpDownCounter`s

Reply:
It's documenting the existing functionality, therefore it should be Gauge, since Gauge is used

Reply:
As part of the wider K8s SemConv stability effort, we can focus on how it should be in SemConv and fix the Collector later. See https://github.com/open-telemetry/semantic-conventions/pull/2267#discussion_r2120289448.